discovery/manager/manager.py

```diff
@@ -354,36 +374,6 @@ def get_log_file_path(input_context: InputContext, service: ServiceData, hosts:
 
         return None
 
-    @staticmethod
```

Comment:
do we now have a new function equivalent to this one? How will we get the values of these env variables now?

Reply:
We have replaced this with AbstractPropertyBuilder.get_service_environment_variable which is more reliable and doesn't truncate the longer strings